devlib/target.py

```diff
@@ -1035,6 +1035,18 @@ def dump_logcat(self, filepath, filter=None, append=False, timeout=30):  # pylin
     def clear_logcat(self):
         adb_command(self.adb_name, 'logcat -c', timeout=30)
 
+    def adb_reboot_bootloader(self):
+        adb_command(self.adb_name, 'reboot-bootloader', timeout=30)
+
+    def adb_root(self):
+        adb_command(self.adb_name, 'root', timeout=30)
```

Comment:
Ok, I've got something working... let's refresh the PR and see if what I'm proposing is sound... otherwise I still have my :parasol_on_ground: to shield from :tomato: s